share/ext/luajit/build_luajit.sh

```diff
@@ -245,17 +245,28 @@ case $1 in
 			export DEFOLD_ARCH="64"
 			export TARGET_CFLAGS=""
 			export XCFLAGS="${COMMON_XCFLAGS}"
-			echo "Building $CONF_TARGET ($DEFOLD_ARCH) with '$TARGET_CFLAGS'"
+			echo "Building $CONF_TARGET ($DEFOLD_ARCH) with '$TARGET_CFLAGS ${XCFLAGS}'"
 			set -e
+			make clean
 			make -j8
 			make install
 			mv $PREFIX/bin/$CONF_TARGET/${TARGET_FILE} $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
 			set +e
 
-			# grab our old 32 bit executable and store it in the host package
+			export DEFOLD_ARCH="32"
+			export XCFLAGS="-DLUAJIT_DISABLE_GC64 ${COMMON_XCFLAGS}"
+
+			export HOST_CC="clang"
+			export HOST_CFLAGS="${COMMON_XCFLAGS} -I."
+			export HOST_ALDFLAGS=""
+			export TARGET_LDFLAGS=""
+
+			echo "Building $CONF_TARGET ($DEFOLD_ARCH) with '$TARGET_CFLAGS ${XCFLAGS}'"
 			set -e
-			tar xvf ${DIR}/luajit-2.1.0-beta3-x86_64-macos.tar.gz
-			cp -v bin/x86_64-macos/luajit-32 $PREFIX/bin/$CONF_TARGET/luajit-32
+			make clean
+			make -j8
+			make install
+			mv $PREFIX/bin/$CONF_TARGET/${TARGET_FILE} $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
```

Comment:
Instead of copying the old x86-macos version (very old), I'm trying to use a build using the `-DLUAJIT_DISABLE_GC64`.

Needs to be verified with a 32 bit device (e.g. android)

Reply:
Also, I think I was confused. Since macOS cannot run 32-bit apps anymore, the luajit-32 already has to be a 64-bit application.